apps/front/src/app/(auth)/register/registerForm/passwordStrength/usePasswordStrength.ts

```diff
@@ -0,0 +1,42 @@
+import { useEffect, useState } from 'react'
+
+const usePasswordStrength = (password: string) => {
+    const [progress, setProgress] = useState(0)
+
+    const getActiveColor = () => {
```

Comment:
active color właściwie to możesz wywołać raz  w tym useEffect i zapisać wynik do stanu

Reply:
Nie wiem czy dobrze zrozumiałem polecenie a wiec nie robię jeszcze resolve converation i możesz rzucić okiem czy jest okej.